opencti-platform/opencti-front/tests_e2e/model/report.pageModel.ts

```diff
@@ -31,4 +31,8 @@ export default class ReportPage {
   getItemFromList(name: string) {
     return this.page.getByRole('link', { name }).first();
   }
+
+  selectAllReports() {
+    return this.page.getByTestId('report-page').getByRole('listitem').getByRole('checkbox').check();
```

Comment:
just a comment, not sure if it will be relevant for your case, but for some of the e2e tests I did, the check() method wasn't working properly for checkboxes, I add to use a click() instead

Reply:
fixed